Policy Compliance Checks: Fix typos and the panel title may be covered by dropdown arrow

eason9487 opened the pull request:
### Changes proposed in this Pull Request:

This PR fixes three typos that were found when testing #1686.

### Detailed test instructions:

Check if the typos are replaced with the correct ones.

### Changelog entry

eason9487 committed:
Fix three typos in the PreLaunchChecklist component

mikkamp reviewed (commented):
Thanks for fixing those typos, just have one comment about the period which should be outside the brackets:
> email or phone number).

![image](https://user-images.githubusercontent.com/11388669/193557455-22bc2d99-661f-4503-9de1-6960efbd7d3f.png)
Just wondering about the text for the contact information. In my browser that renders under the dropdown arrow. Is there a way we can shorten that sentence and maybe move some of it's text into the dropdown text?

How about "Confirm your store's contact information is visible on your website"?

eason9487 committed:
Fix that the panel title is rendered under the button with dropdown arrow.

Address https://github.com/woocommerce/google-listings-and-ads/pull/1708#pullrequestreview-1128158032

eason9487 commented:
@mikkamp, thanks for the review.

> Just wondering about the text for the contact information. In my browser that renders under the dropdown arrow. Is there a way we can shorten that sentence and maybe move some of it's text into the dropdown text?

Considering texts could be in other languages after applying the i18n function, the length of a text may still be too long to fit with the panel's available size, so I adjusted CSS to add the needed space for the dropdown arrow.
![Kapture 2022-10-03 at 19 10 09](https://user-images.githubusercontent.com/17420811/193564357-6501e78e-7e85-4f90-8718-202c5fdf8dbc.gif)

Could you help with a new round of code reviews?

mikkamp commented on js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js:
```diff
@@ -180,7 +180,7 @@ const PreLaunchChecklist = ( props ) => {
 							>
 								{ __(
-									'Allow your customers to contact you for product inquiries by including contact information on your website (i,e, contact us form, business profile link, social media, email or phone number.)',
+									'Allow your customers to contact you for product inquiries by including contact information on your website (i.e. contact us form, business profile link, social media, email or phone number.)',
```
```suggestion
									'Allow your customers to contact you for product inquiries by including contact information on your website (i.e. contact us form, business profile link, social media, email or phone number).',
```

eason9487 replied:
Ah, I missed this part in the previous review comment. Thanks!

mikkamp reviewed (approved):
Thanks for the quick change, that looks good to me the text wraps now to leave space for the arrow.
![image](https://user-images.githubusercontent.com/11388669/193565852-ab0cb80b-a8ca-4568-b6fb-913cec3008ef.png)

I'll go ahead and approve but I still left one suggestion about the period moving outside the brackets.

eason9487 committed:
Move the period outside the brackets.

Address https://github.com/woocommerce/google-listings-and-ads/pull/1708#discussion_r985674170

Co-authored-by: Mik <mikkamp@users.noreply.github.com>
